packages/lesswrong/lib/vulcan-users/permissions.ts

```diff
@@ -114,6 +116,26 @@ export const userOwns = function (user: UsersMinimumInfo|DbUser|null, document:
   }
 };
 
+export const documentIsNotDeleted = (
```

Comment:
Name is maybe a bit misleading

Reply:
Yeah, I wondered about this but couldn't come up with anything better. It seems fine to me in the context of being used alongside `userOwns`, along with it being in `permissions.ts`. Suggestions are welcome though.